test/boost/memtable_test.cc

```diff
@@ -1173,7 +1174,7 @@ SEASTAR_TEST_CASE(flushing_rate_is_reduced_if_compaction_doesnt_keep_up) {
     });
 }
 
-static future<> exceptions_in_flush_helper(std::unique_ptr<sstables::file_io_extension> mep, bool& should_fail, const bool& did_fail) {
+static future<> exceptions_in_flush_helper(std::unique_ptr<sstables::file_io_extension> mep, bool& should_fail, const bool& did_fail, bool expect_isolate = true) {
```

Comment:
since we always pass this parameter to this function, there is no need to use a default value for it, am i right?

Reply:
Sure.